lib/presentation/mixins/connect_page_mixin.dart

```diff
@@ -81,11 +88,16 @@ mixin ConnectPageMixin {
     return '${AppConfig.registrationUrl}?$redirectPublicPlatformOnWeb=$redirectUrlEncode&app=${AppConfig.appParameter}';
   }
 
-  String? _getLogoutUrl(BuildContext context) {
-    final authUrl = Matrix.of(context).authUrl;
-    if (authUrl == null) return null;
-
-    return '$authUrl?logout=1';
+  String _getLogoutUrl({
+    required String authUrl,
+    String? redirectUrl,
+  }) {
+    if (redirectUrl != null) {
+      final redirectUrlEncode = base64Url.encode(utf8.encode(redirectUrl));
```

Comment:
it also was removed. https://github.com/linagora/twake-on-matrix/blob/fce446f8ee8a7e3b31b014ee22ac455ef9bcbfde/lib/presentation/mixins/connect_page_mixin.dart#L84. Please remove it too

Reply:
Remove `redirectUrl`, the browser cannot auto close. Please see: https://github.com/linagora/twake-on-matrix/pull/2337#issuecomment-2885964305

Reply:
Updated